manifests/install.pp

```diff
@@ -1,5 +1,61 @@
-class flapjack::install {
+# = Class: flapjack::install
+#
+# Install FlapJack
+#
+class flapjack::install(
+  $version = 'latest'
```

Comment:
Is there any neat way we can limit this to the latest release in the 1.x.x series? (There are a number of changes in 2.x that mean this module is unlikely to work without significant work).